File: Curso Java/Android/LogYUs/app/src/main/java/com/example/logyus/MainActivity.java
package com.example.logyus;

import androidx.appcompat.app.AppCompatActivity;

import android.annotation.SuppressLint;
import android.os.Bundle;
import android.util.Log;
import android.view.View;
import android.widget.EditText;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void enviar(View view){
         EditText textU = findViewById(R.id.Usuario);
         EditText textP = findViewById(R.id.Pass);
        Log.i("Usuario: ", textU.getText().toString());
        Log.i("Password: ",textP.getText().toString());


    }
}

File: Curso Java/Android/ContadorKotlin/app/src/main/java/com/example/contadorkotlin/MainActivity.kt
package com.example.contadorkotlin

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.TextureView
import android.view.View
import android.widget.TextView
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    var contador = 0

    fun incrementar(view: View){
        this.contador++
        this.text.text = this.contador.toString()
    }

    fun reiniciar(view: View){
        contador = 0
        this.text.text = this.contador.toString()
    }
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        var text = findViewById<TextView>(R.id.text)
       // textView.setText("Holiii")
        //textView.text = "Hola bb"
    }
}